packages/forms-web-app/src/views/layouts/main.njk

```diff
@@ -41,7 +41,7 @@
       {% block backButton %}
         {{ govukBackLink({
           text: "Back",
-          href: "javascript:history.back()",
+          href: navigation[1],
```

Comment:
the middleware ensures there will always be at least two items in the navigation history - the current page, and the fallback page.

There is an edge case where this would break, and that would be if we showed the back button on the fall back page. This should never happen, though if it ever does, it should be caught by this test:

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/919/files#diff-2f8b09958305a61fd225779a562fa8dcd065b8572ae31fe6f072e36282c4fcf1R74